deployment/stacks/parent.ts

```diff
@@ -54,7 +54,7 @@ export class ParentStack extends Stack {
     const vpc = ec2.Vpc.fromLookup(this, "Vpc", { vpcName });
     const privateSubnets = sortSubnets(vpc.privateSubnets).slice(0, 3);
 
-    if (!isDev) {
+    if (!isDev) { // resources that must be in AWS account
```

Comment:
I wonder if we should create a separate prep-the-aws-account stack that is separate from the main cdk stack. That feels more true to what we're really doing here. Same applies to the other commented `!isDev`.

Reply:
I like that idea.